rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-instance-types
+[id="rosa-hcp-instance-types"]
+= ROSA with HCP instance types
+
+All {hcp-title} clusters require a minimum of 2 worker nodes. All {hcp-title} clusters support a maximum of 90 worker nodes. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss.
```

Comment:
replace with `All {hcp-title} clusters require a minimum of 2 worker nodes and support a maximum of 90 worker nodes`

Reply:
`not supported` instead of `unsupported`? as not supported allows for it possibly being supported in the future - just a suggestion

Reply:
Good ideas -fixed!